watson-developer-cloud/discovery-components
# fix: Fix PdfView alignment in DocumentPreview

#### What do these changes do/fix?
Preview content are expected to be aligned center (HtmlView does so), but PdfViewer is left-aligned. This PR is to make PDF page in PdfViewer center-aligned.

First, revise the selector name in commit bb7962d, then insert a wrapper div for PDF page alignment in 13e91a6.. 

![image](https://user-images.githubusercontent.com/19485344/146188098-6216e021-feef-4e41-b840-78a5bb28d5c3.png)


#### How do you test/verify these changes?
Use Storybook. Open DocumentPreview > components > PdfViewer, and check if the page is aligned center

#### Have you documented your changes (if necessary)?
N/A

#### Are there any breaking changes included in this pull request?
No
